python/BioSimSpace/Parameters/Protocol/_amber.py

```diff
@@ -199,18 +325,48 @@ def __init__(self, water_model=None, leap_commands=None, property_map={}):
                     raise TypeError("'leap_commands' must be a 'list' of 'str' types.")
         self._leap_commands = leap_commands
 
+        # Validate the bond records.
```

Comment:
There is some code duplication so could be useful to have a disulfide mixin or something like that.

Reply:
Good idea, I'll give this a go.